framework/update/internal/platform/mac/macupdateinstaller.cpp

```
@@ -0,0 +1,256 @@
+        return RetVal<muse::io::path_t>(make_ret(Err::UnknownError));
+    }
+
+    // 1. Verify the dmg signature before opening it. The unpacked bundle is
+    //    not deep-verified here: the helper re-verifies the staged bundle
+    //    right before the swap.
+    Ret ret = verifyPackageSignature(package);
+    if (!ret) {
+        return RetVal<muse::io::path_t>(ret);
+    }
+
+    const QString stagingDir = configuration()->updateDataPath().toQString() + "/staging";
```

Clang (14.0.6)

[warning] variable 'stagingDir' is not initialized

(cppcoreguidelines-init-variables)